iml-agent/src/action_plugins/stratagem_action_warning.rs

```diff
@@ -2,11 +2,11 @@
 // Use of this source code is governed by a MIT-style
```

Comment:
This should either be within `manage_stratagem.rs` or a new directory `stratagem` should be made that contains both of those files.